static/blockchains/blockchains.json

```diff
@@ -225,11 +304,29 @@
     },
     "folder": "",
     "URL": {
-      "rest": [ "https://rpc.q.org" ],
-      "rpc": [ "https://rpc.q.org" ],
+      "rest": ["https://rpc.q.org"],
+      "rpc": ["https://rpc.q.org"],
       "block": "https://explorer.q.org/block/%block%/transactions/",
       "tx": "https://explorer.q.org/tx/%tx%/",
       "wallet": "https://explorer.q.org/address/%address%/"
     }
+  },
+  "radix": {
+    "name": "Radix",
+    "description": "The only decentralized network that lets you build fast, rewards everyone who makes it better, and scales without friction.",
+    "chain_id": "olympia",
+    "prefix": "rdx",
+    "icon": "https://assets-global.website-files.com/6053f7fca5bf627283b582c2/606d99f3190d3a859603fbb5_r-symbol.svg",
+    "native_tokens": {
+      "denoms": ["xrd_rr1qy5wfsfh"]
+    },
+    "folder": "",
+    "URL": {
+      "rest": [],
+      "rpc": [],
```

Comment:
Add the following URLs:

- https://mainnet.radixdlt.com
- https://gateway.stakesafe.net
- https://api.radix.live
- https://gateway.stakesafe.net
- https://mainnet.clana.io

Part of the list was from: https://www.radixscan.io/CommunityGateways.shtml